source/views/streaming/streams/list.tsx

```diff
@@ -28,6 +29,22 @@ const groupStreams = (entries: StreamType[]) => {
 	return toPairs(grouped).map(([title, data]) => ({title, data}))
 }
 
+const getEnabledCategories = (filters: ListType[]) => {
+	return filters.flatMap((filter: ListType) =>
+		filter.spec.selected.flatMap((spec) => spec.title),
+	)
+}
+
+const filterStreams = (streams: StreamType[], filters: ListType[]) => {
```

Comment:
The type of `filters` (`ListType`) doesn't make sense to me when I am reading this. What is that? Are we filtering lists? Can "lists" be passed as filters? I feel like this is an upstream issue with `@frogpond/filter`, so we can ignore that, but that's why I was confused by this code.

(Also the `Type` suffix should get dropped from types as a general rule. Types are types, yes. And a more TypeScript-y way would have us make these full-fledged interfaces, I think, but that's a wide-scale change.)

Reply:
I want to say that back in Flow, there wasn't as much separation between the "type namespace" and "value namespace", so I tended to suffix things with `Type$`. This is my fault for bad API design. We should fix it upstream, yeah.

Reply:
Tracking this issue at https://github.com/StoDevX/AAO-React-Native/issues/6435